src/Xtools/Pages.php

```diff
@@ -29,6 +29,9 @@ class Pages extends Model
     /** @var string One of 'noredirects', 'onlyredirects' or blank for both. */
     protected $redirects;
 
+    /** @var string One of 'both', 'live' or 'deleted' */
+    protected $deletedPages;
```

Comment:
I would probably just name this property `deleted`, since it doesn't actually contain the pages.